### PR TITLE
Fix bug regarding quantity max integer

### LINKED ISSUE #224: [PE-D] User Guide: Maximum integer value not specified.
The user guide the Quantity when adding an item must be at least 1 (as shown in the image below) but does not specify an upper limit.

The following command adheres to the fact that Quantity must be at least 1 but fails.
```
add n/Grain of rice l/Kitchen q/9999999999999999999 e/2099-02-21
```

Admittedly this is not a bug a normal user might encounter, but I felt that because an upper boundary was not specified in the UG, that perhaps the app's `add` command would not enforce any upper boundary.

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: douglaswja/ped#1

### LINKED ISSUE #259: [PE-D] Maximum Quantity value not stated
When trying to add a large quantity to an item, an error message of maximum value exceeded. 

There is no mention of maximum quantity.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: justgnohUG/ped#5

### PR DESCRIPTION
**Pull request by kumsssss**
1) Set the upper limit for quantity to be 1,000,000
2) Included this constraint in the UG.

Closes #224 
Closes #259

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -100,13 +100,13 @@ StoreMando does not allow two identical items to exist.
...
 --------------------| -------
-Item Name  | An alphanumeric string that can contain spaces.
-Location | A string that can contain spaces and symbols.
-Quantity | An integer that must be at least 1.
+Item Name  | An alphanumeric string that can be one word or more.
+Location | A string.
+Quantity | An integer that must be at least 1 and cannot exceed 1,000,000.
 Expiry Date | A date in the format of YYYY-MM-DD. An item can either have 1 expiry date or no expiry date.
-Tag | An alphanumeric string that cannot contain spaces. An item can 0 or more tags.
+Tag | An alphanumeric string that can only be one word long. An item can 0 or more tags.
```
Comment (Md-Fazil) [inline docs/UserGuide.md]: Should this be "An item can have 0 or more tags."?

**Review by Md-Fazil — COMMENTED** [review]
Some nits and grits to fix before merging.

File: docs/UserGuide.md (hunk 1 above)
Comment (kumsssss) [reply docs/UserGuide.md]: yes i changed it. thx!

**Review by JayChenYJ — APPROVED** [review]
LGTM!

**Pull request merged**